### DISCUSSION TITLE
posts/Go/grpc-gateway/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#311] [Announcements]
https://www.liwenzhou.com/posts/Go/grpc-gateway/ 

 李文周的Blog提供免费的Go语言教程，本文是gRPC-Gateway的中文使用指南，演示了如何安装 gRPC-Gateway插件，通过为我们的protobuf文件添加注释从而为我们的RPC服务自动生成RESTful API代码。还详细介绍了如何在protobuf中导入google api的annotations.proto文件。

**Comment by TreeZeng** [comment]
niubi

**Comment by s-z-z** [comment]
client代码可以使用gateway生成的client, 这样不用记url和json结构
